source/agora/node/Ledger.d

```diff
@@ -518,8 +522,8 @@ public class Ledger
 
     public string validateSlashingData (in ConsensusData data) @safe
     {
-        return this.slash_man.isInvalidPreimageRootReason(this.getBlockHeight(),
-                data.missing_validators);
+        return this.slash_man.isInvalidPreimageRootReason(
+                this.last_block.header.height, data.missing_validators);
```

Comment:
Is this change just an optimization? Or is the height passed here different now?

Reply:
It changed unintentionally. There's no difference between before and after. I'll restore it.